src/App.tsx

```diff
@@ -14,37 +13,111 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type TypeIsReversed = {
+  isReversed: boolean;
+};
+
+type TypeSort = {
+  sortType: number;
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { isReversed }: TypeIsReversed,
+  { sortType }: TypeSort,
+) {
+  const visibleGoods = [...goods];
+
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort((a, b) => (a > b ? 1 : -1));
```

Comment:
using localeCompare method is more safe

Reply:
localeCompare doesn't work. 

`      visibleGoods.sort((a, b) => a.localeCompare(b));`